system/include/emscripten/em_js.h

```diff
@@ -9,14 +9,6 @@
 
 #include <emscripten/em_macros.h>
 
-#ifdef __cplusplus
-#define _EM_JS_CPP_BEGIN extern "C" {
-#define _EM_JS_CPP_END   }
-#else // __cplusplus
-#define _EM_JS_CPP_BEGIN
-#define _EM_JS_CPP_END
-#endif // __cplusplus
```

Comment:
I wonder if we should keep this begin/end pair?  In order to keep this change small/focused?

Changing from `extern "C" { int foo }` to  `extern "C" foo`  is a breaking change, for example.

But I get that this is only used in specific places where the variable has an initializer so maybe its fine.

Reply:
Yeah I mean this is purely internal macro that only used to have 1 and now 2 usages. It's even prefixed with `_EM_JS*`, so it was never meant as a general-purpose `extern "C" {` wrapper.

Reply:
Narrator: this was wrong. https://github.com/emscripten-core/emscripten/pull/20346